File: hw/palomo/u3/hw20FarmAnimalSystem/FarmAnimalSystem/src/main/java/ec/espe/edu/farmanimalsystem/model/Product.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.espe.edu.farmanimalsystem.model;

/**
 *
 * @author Cristian
 */
public class Product {
    private int id;
    private String unit;
    private float quantity;

    public Product(int id, String unit, float quantity) {
        this.id = id;
        this.unit = unit;
        this.quantity = quantity;
    }

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the unit
     */
    public String getUnit() {
        return unit;
    }

    /**
     * @param unit the unit to set
     */
    public void setUnit(String unit) {
        this.unit = unit;
    }

    /**
     * @return the quantity
     */
    public float getQuantity() {
        return quantity;
    }

    /**
     * @param quantity the quantity to set
     */
    public void setQuantity(float quantity) {
        this.quantity = quantity;
    }

    @Override
    public String toString() {
        return "Product{" + "id=" + id + ", unit=" + unit + ", quantity=" + quantity + '}';
    }
    
}

File: hw/tipantiza/u3/hw21FarmSystemPOO/FarmSystem_py/model/product.py
class Product:
    def __init__(self, id: int, description: str, unit: str, quantity: float):
        self.id = id
        self.description = description
        self.unit = unit
        self.quantity = quantity
    
    def __str__(self):
        return f"Product(id={self.id}, description={self.description}, unit={self.unit}, quantity={self.quantity})"
    
    def get_id(self) -> int:
        return self.id
    
    def set_id(self, id: int):
        self.id = id
    
    def get_description(self) -> str:
        return self.description
    
    def set_description(self, description: str):
        self.description = description
    
    def get_unit(self) -> str:
        return self.unit
    
    def set_unit(self, unit: str):
        self.unit = unit
    
    def get_quantity(self) -> float:
        return self.quantity
    
    def set_quantity(self, quantity: float):
        self.quantity = quantity